qubesbuilder/distribution.py

```diff
@@ -0,0 +1,94 @@
+# The Qubes OS Project, http://www.qubes-os.org
+#
+# Copyright (C) 2021 Frédéric Pierret (fepitre) <frederic@invisiblethingslab.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import re
+
+from qubesbuilder.exc import DistributionError
+
+FEDORA_RE = re.compile("^fc([0-9]+)$")
+CENTOS_STREAM_RE = re.compile("^centos-stream([0-9]+)$")
+
+DEBIAN = {
+    "stretch": "9",
```

Comment:
```suggestion
```

Debian Stretch is EOL, so there is no need to mention it.

Reply:
https://www.debian.org/releases/ says stretch is still under LTS.